drake/solvers/trajectoryOptimization/CGL_nodes.m

```diff
@@ -0,0 +1,30 @@
+%--------------------------------------------------------------------------
+% CGL_nodes.m
+% determines Chebyshev-Gauss-Lobatto (CGL) nodes
+%--------------------------------------------------------------------------
+% tau = CGL_nodes(N)
+%   N: number of nodes minus 1, should be an integer greater than 0
+% tau: CGL nodes
+%--------------------------------------------------------------------------
+% Examples:
+% tau = CGL_nodes(1)
+% -1     1
+% tau = CGL_nodes(2)
+% -1     0     1
+% tau = CGL_nodes(3)
+% -1  -0.5   0.5   1
+%--------------------------------------------------------------------------
+% Author: Daniel R. Herber, Graduate Student, University of Illinois at
+% Urbana-Champaign
+% Date: 06/04/2015
+%-------------------------------------------------------------------------
+function tau = CGL_nodes(obj)

```

Comment:
This function uses symbolic computation first, and then substitute the numerical values. Consider to rewrite it with numeric computation directly.